snmp/tests/test_e2e_core_profiles/test_profile_apc_netbotz.py

```diff
@@ -362,7 +362,7 @@ def test_e2e_profile_apc_netbotz(dd_agent_check):
             'netbotz_error_cond_enc_id:quaintly quaintly acted their',
             'netbotz_error_cond_id:their acted forward forward their',
             'netbotz_error_cond_index:2828061222',
-            'netbotz_error_cond_resolved:none',
+            'netbotz_error_cond_resolved:',
```

Comment:
That's strange, we have no value. Is that because we are using `null` here https://github.com/DataDog/integrations-core/pull/15668/files#diff-785bedfa456ffb2c5737fa96776ad24fef1e6a4e7a31ccef0c32fe896725b1f3R355

Reply:
Related to https://github.com/DataDog/integrations-core/pull/15668/files/97c5f789f8a078604ddb7e1f77b909da9741ef6c#r1307607195